packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -431,50 +456,68 @@ export class AzureCommunicationCallWithChatAdapter implements CallWithChatAdapte
       attachmentMetadata?: AttachmentMetadata[];
     }
   ): Promise<void> {
-    return await this.chatAdapter.updateMessage(
-      messageId,
-      content,
-      metadata,
-      /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */ options
-    );
+    return this.chatAdapterPromise.then((adapter) => {
+      return adapter.updateMessage(
+        messageId,
+        content,
+        metadata,
+        /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */ options
+      );
+    });
   }
   /** Delete an existing message. */
   public async deleteMessage(messageId: string): Promise<void> {
-    return await this.chatAdapter.deleteMessage(messageId);
+    return await this.chatAdapterPromise.then((adapter) => {
+      return adapter.deleteMessage(messageId);
+    });
   }
   /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
   public registerActiveFileUploads = (files: File[]): FileUploadManager[] => {
-    return this.chatAdapter.registerActiveFileUploads(files);
+    return this.chatAdapter?.registerActiveFileUploads(files) ?? [];
```

Comment:
This should throw an error if called before the adapter is initialized, otherwise this will silently fail for any developer calling this before the adapter is initialized